src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,20 @@
-export const TodoInfo = () => {};
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Types';
+import classNames from 'classnames';
+import React from 'react';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    data-id={todo.id}
+    className={classNames('TodoInfo', {
+      'TodoInfo--completed': todo.completed,
+    })}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+    {todo && <UserInfo user={todo.user} />}
```

Comment:
The condition `todo && <UserInfo user={todo.user} />` checks if `todo` is defined, but it does not explicitly check if `todo.user` is defined. Consider adding a check to ensure `todo.user` is not undefined to prevent potential runtime errors.